help/sources/tutorials/api/create/analytics/mixpanel.md

```diff
@@ -0,0 +1,1072 @@
+                "mp_processing_time_ms": 1648480685058
+            }
+        },
+        {
+            "event": "Item Purchased",
+            "properties": {
+                "time": 1648165814,
+                "distinct_id": "test1@test.com",
+                "$insert_id": "935d87b1-00cd-41b7-be34-b9d98dd08b75",
+                "$mp_api_endpoint": "api.mixpanel.com",
+                "$mp_api_timestamp_ms": 1648551687866,
+                "item_id": "29073",
+                "item_name": "Watch",
+                "item_price": "50000.00",
+                "mp_processing_time_ms": 1648551687922
+            }
+        },
+        {
+            "event": "Sign off",
+            "properties": {
+                "time": 1648530419,
+                "distinct_id": "test1@test.com",
+                "$insert_id": "935d87b1-00cd-41b7-be34-b9d98dd08b75",
+                "$mp_api_endpoint": "api.mixpanel.com",
+                "$mp_api_timestamp_ms": 1648555619274,
+                "item_id": "29073",
+                "item_name": "Watch",
+                "item_price": "50000.00",
+                "mp_processing_time_ms": 1648555619326
+            }
+        },
+        {
+            "event": "Items sold",
+            "properties": {
+                "time": 1648566534,
+                "distinct_id": "test1@test.com",
+                "$import": true,
+                "$insert_id": "935d87b1-00cd-41b7-be34-b9d98dd08b75",
+                "$mp_api_endpoint": "api.mixpanel.com",
+                "$mp_api_timestamp_ms": 1648635830114,
+                "item_id": "29073",
+                "item_name": "Pen",
+                "item_price": "1000.00",
+                "mp_processing_time_ms": 1648635831010
+            }
+        }
+    ]
+}
+```
+
+### Create a source connection {#source-connection}
+
+You can create a source connection by making a POST request to the [!DNL Flow Service] API. A source connection consists of a connection ID, a path to the source data file, and a connection spec ID.
+
+**API format**
+
+```https
+POST /sourceConnections
+```
+
+**Request**
+
+The following request creates a source connection for *Mixpanel*:
+
+```shell
+curl -X POST \
+    'https://platform.adobe.io/data/foundation/flowservice/sourceConnections' \
+    -H 'Authorization: Bearer {ACCESS_TOKEN}' \
+    -H 'x-api-key: {API_KEY}' \
+    -H 'x-gw-ims-org-id: {ORG_ID}' \
+    -H 'x-sandbox-name: {SANDBOX_NAME}' \
+    -H 'Content-Type: application/json' \
+    -d '{
+        "name": "{Mixpanel} Source Connection",
+        "description": "{Mixpanel} Source Connection",
+        "baseConnectionId": "70383d02-2777-4be7-a309-9dd6eea1b46d",
+        "connectionSpec": {
+            "id": "6360f136-5980-4111-8bdf-15d29eab3b5a",
+            "version": "1.0"
+        },
+        "data": {
+            "format": "json"
+        },
+        "params": {
+            "projectId": "{MIXPANEL_PROJECT_ID}",
+            "timezone": "{MIXPANEL_PROJECT_ID}"
```

Comment:
typo MIXPANEL_PROJECT_ID should be MIXPANEL_TIMEZONE

Reply:
Oops, good spot.